From 788635a991adfa91f2fa48e59ac78f405ee135df Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 10 Oct 2024 02:01:06 +0200
Subject: [PATCH] Look for assets in the right place

---
 NiimPrintX/ui/__main__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/NiimPrintX/ui/__main__.py b/NiimPrintX/ui/__main__.py
index 535470ac4356..573203b0fbe0 100644
--- a/NiimPrintX/ui/__main__.py
+++ b/NiimPrintX/ui/__main__.py
@@ -32,7 +32,7 @@ def resource_path(relative_path):
     if hasattr(sys, '_MEIPASS'):
         base_path = getattr(sys, '_MEIPASS', os.path.dirname(os.path.abspath(__file__)))
     else:
-        base_path = os.path.realpath(".")
+        base_path = os.path.realpath("/app/lib/python3.13/site-packages")
 
     return os.path.realpath(os.path.join(base_path, relative_path))
 
-- 
2.54.0

